src/offset/mod.rs

```diff
@@ -269,7 +263,8 @@ pub trait TimeZone: Sized + Clone {
     /// but it will propagate to the `DateTime` values constructed via this date.
     ///
     /// Panics on the out-of-range date and/or invalid DOY.
-    #[deprecated(since = "0.4.23", note = "use `ymd_opt()` instead")]
+    #[deprecated(since = "0.4.23", note = "use `from_local_datetime()` with a `NaiveDateTime` instead")]
```

Comment:
we could reccomend `.and_timezone()` instead of `.from_local_datetime()` here and in similar places

Reply:
except on probably `from_local_date` as there `from_local_datetime` is the natural choice

Reply:
Yup, good call.